src/app/submission/components/submit/submit.component.ts

```diff
@@ -81,9 +81,7 @@ export class SubmitComponent implements OnInit {
       },
       err => {
         this.loaderService.display(false);
-        this.alertService.error('', 'An error occurred on submitting your submission envelope.', true);
-        console.log(err);
-
+        this.alertService.error('An error occurred on submitting your submission envelope.', err.message, true);
```

Comment:
i think the keepAfterRouteChange param in the `alertService.error` function should be false